packages/cryptography/meta.yaml

```diff
@@ -1,18 +1,32 @@
 package:
   name: cryptography
-  version: 3.4.8
+  version: 36.0.2
 
 source:
-  url: https://files.pythonhosted.org/packages/cc/98/8a258ab4787e6f835d350639792527d2eb7946ff9fc0caca9c3f4cf5dcfe/cryptography-3.4.8.tar.gz
-  sha256: 94cc5ed4ceaefcbe5bf38c8fba6a21fc1d365bb8fb826ea1688e3370b2e24a1c
+  url: https://files.pythonhosted.org/packages/10/a7/51953e73828deef2b58ba1604de9167843ee9cd4185d8aaffcb45dd1932d/cryptography-36.0.2.tar.gz
+  sha256: 70f8f4f7bb2ac9f340655cbac89d68c527af5bb4387522a8413e841e3e6628c9
+  patches:
+    - patches/0001-Use-patched-chrono.patch
 build:
+  script: |
+    source $HOME/.cargo/env
+    rustup update nightly
+    rustup component add rust-src --toolchain nightly-x86_64-unknown-linux-gnu
+    export CARGO_BUILD_TARGET=wasm32-unknown-emscripten
+    export PYO3_CONFIG_FILE=$PKG_DIR/pyo3_config.ini
+    CARGO_TARGET_WASM32_UNKNOWN_EMSCRIPTEN_LINKER="emcc"
+    export EMSCRIPTEN_LINKING_RUST=1
+    export RUSTFLAGS=" \
+      -C relocation-model=pic \
+      -C target-feature=+mutable-globals \
+      -C link-arg=-sSIDE_MODULE=1 \
+    "
+    git clone --depth 1 https://github.com/hoodmane/chrono.git --branch pyodide
```

Comment:
Maybe we could add a TODO comment to remove this one https://github.com/chronotope/chrono/pull/593 is released.